src/services/catalog.service.ts

```diff
@@ -127,7 +127,7 @@ export class ServiceItem implements IServiceItem {
   }
 
   private getDescription() {
-    return _.get(this.resource, 'osbMetadata.description', '');
+    return _.get(this.resource, 'description', '');
```

Comment:
Yeah, I didn't realize the mock data was wrong. It should be a top-level item, not under `osbMetadata`.